src/components/MovedNodeCard.tsx

```diff
@@ -74,44 +42,33 @@ interface Props {
     url?: string;
     title?: string;
   }[];
+  breadcrumbs?: string[];
 }
 
-export const MovedNodeCard = ({ title, url, ingress, subjects, contentType, metaImage }: Props) => {
-  const urlLabelId = useId();
+export const MovedNodeCard = ({ title, url, ingress, breadcrumbs, contentType, metaImage }: Props) => {
   const { t } = useTranslation();
 
   return (
-    <StyledCardRoot>
-      <Wrapper>
+    <>
+      <StyledCardRoot>
+        {metaImage && metaImage.url && <CardImage alt={metaImage.alt ?? ""} src={metaImage.url} />}
         <CardContent>
-          {!!contentType && <ContentTypeBadgeNew contentType={contentType} />}
+          <ContentTypeBadgeNew contentType={contentType}>{t(`contentTypes.${contentType}`)}</ContentTypeBadgeNew>
           <CardHeading asChild consumeCss>
-            <h2>
-              <SafeLink to={url} css={linkOverlay.raw()}>
+            <h3>
```

Comment:
Trenger vi egentlig en tittel her i det hele tatt? Kan vi asChilde rett på SafeLink?

Reply:
Ikke for min del, dette er stjålet fra et annet sted så jeg bare antok det var sånn semantikken burde være :shrug: